modules/form-field/react/spec/__snapshots__/Label.snapshot.tsx.snap

```diff
@@ -26,10 +26,9 @@ exports[`Label Snapshots renders left label 1`] = `
   font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
   padding: 0;
   display: inline-block;
-  vertical-align: top;
-  margin-top: 10px;
```

Comment:
This create an issue though. On `Checkbox` and `Switch`, the `Label`, when on the left, is vertically centered. But on other components, it’s not.

One way to fix this would be to restore `marginTop: 10` but it will therefore push the Label to the bottom for Checkbox and Switch as well.

We might need a way to position the Label vertically based on the component that is on the right, any thoughts?

You can see this on that temporary story that will be removed after we find a solution: http://localhost:9001/?path=/story/form-field-test--default